scenarios/kubernetes_e2e.py

```diff
@@ -445,8 +449,7 @@ def create_parser():
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
     parser.add_argument(
-        '--multiple-federations', default=False, action='store_true',
-        help='If we need to run multiple federation control planes in parallel')
+        '--user', default='prow', help='Username used for ssh in local mode')
```

Comment:
The only thing that is wrong before is that the user was root and so when we ran config-ssh we uploaded the ssh key for user root, but the VM does not allow user root to ssh